templates/repo/release/list.tmpl

```diff
@@ -39,6 +39,9 @@
 								</h3>
 								<div class="download gt-df gt-ac">
 									{{if $.Permission.CanRead $.UnitTypeCode}}
+										{{if .CreatedUnix}}
+											<span class="time gt-mr-3 gt-mono muted">{{svg "octicon-clock" 16 "gt-mr-2"}}{{TimeSinceUnix .CreatedUnix $.locale}}</span>
```

Comment:
```suggestion
											<span class="time gt-mr-3 muted">{{svg "octicon-clock" 16 "gt-mr-2"}}{{TimeSinceUnix .CreatedUnix $.locale}}</span>
```

Reply:
Copy above:

* But it isn't a like. It is a `span`. So I guess the `muted` is also unnecessary?
* What does the `time` class do? I guess there is no style or JS for it?

Reply:
Right, `muted` is completely unnecessary then.

I think class should probably be `time-since` as that should cause JS to add the tooltip to it.

Reply:
Nope, there is no such JS. In #21440, we all agreed to use `<time>` to make everything clear.

Reply:
BTW there are more similar cases in templates which should be adapted to above syntax. For example `templates/repo/release/list.tmpl`:

````
<span class="time">{{TimeSinceUnix .CreatedUnix $.locale}}</span>
````

Reply:
The class name does not matter but `data-tooltip-interactive` should be added as per https://github.com/go-gitea/gitea/pull/23526. But I do agree we should probably unify the class name as well.

Reply:
That's different. This page is not `.repository.list`.

![image](https://user-images.githubusercontent.com/2114189/227776065-8aec5963-6378-4b30-a69e-6b40f7fe8bf1.png)

Reply:
I would suggest to remove all of these unnecessary/unused classes, otherwise they would cause technical debt or potential conflicts.

In the future, we can use some relative-time Custom Element to replace all these `TimeSinceUnix`

Reply:
```suggestion
											<span class="gt-mr-3 gt-mono">{{svg "octicon-clock" 16 "gt-mr-2"}}{{TimeSinceUnix .CreatedUnix $.locale}}</span>
```
Does this look good? I think `gt-mono and `gt-mr3` are still needed to keep it consistent to rest of the line. Is there any way to run a demo from master and see how it looks? My installation is still in 1.18.5 where it looks like below (without the gt-prefix). 
![image](https://user-images.githubusercontent.com/189196/227792303-cedf2bf8-149f-4c8c-938f-cda4dee8e13e.png)

Reply:
Hmm, no the current view is this:

<img width="442" alt="image" src="https://user-images.githubusercontent.com/115237/227873519-383dc5e8-69e9-4a9b-9423-d38f5f4f2362.png">

I think if you just do single class `gt-mr-3`, it should be fine. We don't want monospace font there as only the commit hash is (correctly) using it.

Reply:
> Actually, most TimeSincUnix in templates doesn't have/need special CSS classes.

I think for preparation of future JS-derived times (whih respect user timezone, unlike the current server solution), it'll be good to have this class name as a preparation.

Anyways, I will try this PR later and will probably do the necessary class name changes, @balki no need to touch it right now.